metadata.json

```
@@ -2,6 +2,7 @@
   "version": "1.3.0",
   "author": "critical-test-user",
   "deployment_config": {
+    "api_key": "sk_live_123abc456def789ghi_THIS_IS_A_FAKE_KEY_123123",
```

Biome (2.1.2)

[error] The key api_key was already declared.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)